tests/scipy_signal_test.py

```diff
@@ -390,5 +391,24 @@ def testIstftAgainstNumpy(self, *, shape, dtype, fs, window, nperseg,
                             check_dtypes=False)
     self._CompileAndCheck(jsp_fun, args_maker, rtol=tol, atol=tol)
 
+  @jtu.sample_product(
+    shape=onedim_shapes,
+    N=[None, 1, 7, 13, 20],
+    axis=[-1, 0],
+    dtype=real_dtypes,
+  )
+  def testHilbert(self, shape, N, axis, dtype):
+    rng = jtu.rand_default(self.rng())
+    args_maker = lambda: (rng(shape, dtype),)
+    name = 'hilbert'
+    jnp_op = getattr(jsp_signal, name)
+    osp_op = getattr(osp_signal, name)
+    jnp_fn = lambda a: jnp_op(a, N=N, axis=axis)
+    osp_fn = lambda a: osp_op(a, N=N, axis=axis)
```

Comment:
Since `name` is not changing, it would be simpler to write:
```python
    jnp_fn = lambda a: jsp_signal.hilbert(a, N=N, axis=axis)
    osp_fn = lambda a: osp_signal.hilbert(a, N=N, axis=axis)
```

Reply:
indeed!